src/CMakeLists.txt

```diff
@@ -891,7 +891,6 @@ list( APPEND SOURCES
       widgets/TimeFormat.h
       widgets/UnwritableLocationErrorDialog.cpp
       widgets/UnwritableLocationErrorDialog.h
-      widgets/VetoDialogHook.h
```

Comment:
You are correct that `VetoDialogHook.h` existed only to enable screenshots.  This commit looks complete to me, for the purpose of removing it.